README.md

```diff
@@ -67,6 +67,19 @@ Slime.render(source, site_title: "Website Title")
 
 ## Reference
 
+### Tags
+
+Starting a line with a string followed by a space will create an html tag, as follows:
+
+```slim
+tt
+  Always bring a towel.
+```
+
+```html
+<tt>Always bring a towel.<tt>
```

Comment:
Shouldn't this be either

`"<tt><Always>bring a towel.</Always></tt>"`

or 

```
tt
   | Always bring a towel.
```

?

Reply:
Yeah, you're right. I'll add a suggestion for that, too.

```
iex(1)> Slime.Renderer.precompile("tt\n  Always bring a towel")
"<tt><Always>bring a towel</Always></tt>"
iex(2)> Slime.Renderer.precompile("tt\n  | Always bring a towel") 
"<tt>Always bring a towel</tt>"
```